src/Processors/Content/Save.php

```diff
@@ -321,6 +284,26 @@ private function doNullEdit( User $user, WikiPage $wikiPageObject, $content ) {
 		}
 	}
 
+	/**
+	 * If a form has the permissions argument, then do an additional rebuild data for the page created/edited.
+	 * In some cases a user might not have the correct rights to create a page where a template uses a parser to
+	 * read or set properties. This rebuild data will force to set those properties.
+	 * @param string $title
+	 *
+	 * @return void
+	 */
+	private function forceRebuildDataForTitle( string $title ): void {
+		if ( !ExtensionRegistry::getInstance()->isLoaded( 'SemanticMediaWiki' ) ) {
+			return;
+		}
+		if ( $this->needRebuildData ) {
+			global $IP;
+			$cmd = 'php ' . $IP . '/extensions/SemanticMediaWiki/maintenance/rebuildData.php';
+			$cmd .= ' --page="' . escapeshellarg( $title ) . '"' . ' > /dev/null &';
```

Comment:
`escapeshellarg` already adds single quotes around the string it is given, and therefore the additional double quotes will cause problems.